### DISCUSSION TITLE
No puedo hacer push, me sale error

### DISCUSSION THREAD

**Discussion by Nikzza** [INGCOM-UNRN-P1/cursada-2023#21] [Preguntas y respuestas] [answered]
![Captura de pantalla 2023-08-10 002353](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/141664052/978941cf-10da-4475-bebf-ee06bbbc40e7)
Hola, no puedo utilizar push, me aparece un error cuando quiero enviar el compilador, me habia olvidado de mandarlo y cuando lo quise mandar me apareció esto,

**Comment by martinvilu** [comment, marked as answer]
`git push` no necesita argumentos, debiera de funcionar ejecutado dentro del repositorio

**Reply by Nikzza** [reply]
me aparece everything to up date pero no sube nada jsjs

**Reply by martinvilu** [reply]
`git status` que dice?

Si marca cambios, entonces falta hacer el `git add`/`git commit`

**Reply by Nikzza** [reply]
On branch main
Your branch is up to date with "origin/main"

Changes to be committed: 
(use "git restore --staged <file>... " to unstage)
new file: compilator.txt

**Reply by Nikzza** [reply]
eso me aparece

**Reply by martinvilu** [reply]
efectivamente, falta `git commit`
